common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -141,33 +117,15 @@ object UrlHelpers {
 
   object oldNav {
```

Comment:
Since this is called `oldNav`, am I right in thinking that all of this, plus all the templates using them, will go when the switch the deleted?

Reply:
Yes :). I don't want to remove it now just in case